rules/windows/builtin/win_mal_creddumper.yml

```diff
@@ -13,6 +12,26 @@ tags:
     - attack.t1003
     - attack.t1035
     - attack.s0005
+logsource:
+    product: windows
+    service: system
+detection:
+    selection:
+        EventID: 7045
+    keywords:
+        Message:
+          - '*WCE SERVICE*'
+          - '*WCESERVICE*'
+          - '*DumpSvc*'
+    quarkspwdump:
+        EventID: 16
+        HiveName: '*\AppData\Local\Temp\SAM*.dmp'
+    condition: ( selection and keywords ) or quarkspwdump
```

Comment:
the listed service names:

```
- '*WCE SERVICE*'
- '*WCESERVICE*'
- '*DumpSvc*'
```

have been added (and still present) in `selection_1` fo this rule.

`quarkspwdump` section has been moved to separate [rule](https://github.com/Neo23x0/sigma/blob/master/rules/windows/builtin/win_susp_sam_dump.yml) as it is not directly connected to service execution (according to name and description of the rule), and have been already pushed in `master` branch.

Reply:
Removed quarkspwdump part from creddumper rule.